nautobot/ui/config-overrides.js

```diff
@@ -29,9 +29,10 @@ module.exports = function override(config) {
   // config.output.path = path.resolve(__dirname, "build/static")  // Should be in STATICFILES_DIRS
   // config.output.publicPath = process.env.NAUTOBOT_STATIC_URL // Should match STATIC_URL
   config.output.filename =  "static/js/[name].js" // No filename hashing; Django collectstatic takes care of this
-  config.output.assetModuleFilename = "static/media/[name].[ext]",
+  config.output.assetModuleFilename = "static/media/[name].[ext]"
   config.output.chunkFilename = "static/js/[id]-[chunkhash].js" // DO have Webpack hash chunk filename
 
+  config.devtool = 'eval-cheap-module-source-map'
```

Comment:
This is required to make the devtool output _useful_. Will want to toggle this for production because it's slower to build.

**Before**

<img width="1492" alt="Screenshot 2023-03-06 at 20 10 16" src="https://user-images.githubusercontent.com/31187/223301637-3c8f6ee5-04ab-4597-8949-51ab826169e1.png">

**After**

<img width="1492" alt="Screenshot 2023-03-06 at 20 09 05" src="https://user-images.githubusercontent.com/31187/223301677-69cd5f0a-ed7a-43fe-a078-c400b39042e5.png">

Reply:
https://webpack.js.org/configuration/devtool/

Reply:
Added a  TODO comment to the file.